packages/mdc-drawer/_mixins.scss

```diff
@@ -14,98 +14,89 @@
 // limitations under the License.
 //
 
-@import "@material/animation/functions";
-@import "@material/animation/variables";
 @import "@material/theme/mixins";
-@import "@material/theme/variables";
-@import "@material/typography/mixins";
-
-@mixin mdc-drawer-toolbar-spacer_() {
-  .mdc-drawer__toolbar-spacer {
-    display: flex;
-    position: relative;
-    flex-direction: row;
-    flex-shrink: 0;
-    align-items: center;
-    box-sizing: border-box;
-    height: 56px;
-    padding: 16px;
-    border-bottom: 1px solid rgba(0, 0, 0, .12);
-
-    // TODO(sgomes): replace with global breakpoints when we have them
-    @media (min-width: 600px) {
-      height: 64px;
-    }
+@import "@material/list/mixins";
+@import "@material/ripple/mixins";
+
+@mixin mdc-drawer-active-overlay-color($color) {
+  :not(.mdc-list--non-interactive) > .mdc-list-item {
+    @include mdc-states-activated($color);
+    @include mdc-states-selected($color);
   }
 }
 
-@mixin mdc-drawer-header_() {
-  // Use aspect ratio trick to maintain 16:9 aspect ratio on the header
-  // styline things header is a type
-  // stylelint-disable selector-max-type
-  .mdc-drawer__header {
-    position: relative;
-  }
+@mixin mdc-drawer-border-color($color, $opacity) {
+  $value: rgba(mdc-theme-prop-value($color), $opacity);
 
-  .mdc-drawer__header::before {
-    display: block;
-    padding-top: 9 / 16 * 100%;
-    content: "";
-  }
+  @include mdc-theme-prop(border-color, $value);
+}
 
-  .mdc-drawer__header-content {
-    display: flex;
-    position: absolute;
-    top: 0;
-    right: 0;
-    bottom: 0;
-    left: 0;
-    align-items: flex-end;
-    box-sizing: border-box;
-    padding: 16px;
-  }
-  // stylelint-enable selector-max-type
+@mixin mdc-drawer-divider-color($color, $opacity) {
```

Comment:
OOC, is there a reason why we're wrapping a bunch of list mixins here rather than making the list mixins support opacity?

Reply:
I think I did this after reading the opacity doc. I was thinking that we would eventually do list and other components in a component wide change. I think soon after I did it this way designers were going to take it back to the drawing board.

Reply:
So I guess part of this was to localize the changes to drawer rather than affecting list right away... But I'm wondering whether some of these mixins even need to exist, rather than simply using list mixins directly, if they're doing nothing but calling the list mixins in the long run?

Could we just cross-link to the list mixins in the drawer documentation and cut a bunch of these mixins, presuming we eventually update the list mixins appropriately once we know how we want to handle opacity?

Reply:
So remove all the drawer/list mixins? I think most of the mixins are specific to drawer w/ the exception of the `divider`, `meta`, and `graphic`. It would be strange to remove divider, meta, and graphic, but still have the others proxied from the drawer mixin files.

I do however like the primary-ink-color mixin changing both meta and graphic, but then having the other 2 mixins to override the meta and grahpic.

Reply:
I was wondering if that was the angle you were coming from. I guess that's fair from a perspective of having all relevant customizations in one place, and the fact they really just proxy is an implementation detail.

I suppose the upside to this is it gives us the freedom to tweak it uniquely from list if we need to, while the downside is if we ever need to make breaking changes to list APIs, we'll need to be careful to update drawer and menu as well. (Hopefully we can add screenshot tests for these revamped packages SRTL to help mitigate this.)